lib/translations.json

```diff
@@ -3177,6 +3177,26 @@
     {
       "key": "styleguide/DynamicComponent/error",
       "value": "Diese Komponente kann, aufgrund eines technischen Fehlers, nicht dargestellt werden."
+    },
+    {
+      "key": "pledge/fields/explanation",
+      "value": "Fast geschafft. Ihr Zugang ist ein Konto erster Güte, und Sie können unter anderem an Debatten teilnehmen. Um Ihr Profil für den Start zu vervollständigen, benötigen wir noch folgende Informationen von Ihnen:"
```

Comment:
Use `tokenAuthorization` name space.

I would avoid wording like «Fast geschafft.», sign in is only the start :)

I'd go with some more confident and assertive:
```
Um Ihr Konto zu aktivieren, benötigen wir folgende Informationen von Ihnen:
```